Gordon360/Models/ViewModels/ApartmentApplicationViewModel.cs

```diff
@@ -23,7 +23,7 @@ public string EditorUsername
         public List<ApartmentApplicantViewModel> Applicants { get; set; }
         public List<ApartmentChoiceViewModel> ApartmentChoices { get; set; }
         public int TotalPoints { get { return Applicants?.Sum(applicant => applicant.Points) ?? 0; } }
-        public double AvgPoints { get { return Applicants?.Average(applicant => applicant.Points) ?? 0; } }
+        public double AvgPoints { get { return Math.Round(Applicants?.Average(applicant => applicant.Points) ?? 0, 2); } }
```

Comment:
This seems like it belongs more in the formatting logic on the frontend - since it's inherently a display rule. I'm not sure it should really be enforced here.

Reply:
I can't think of a case where we would want that level of detail on the average, so I included it in the API.